SingularityUI/app/components/common/table/UITable.jsx

```diff
@@ -229,12 +235,13 @@ class UITable extends Component {
     const rowsPerPage = this.props.rowChunkSize;
     if (this.props.paginated && numRows > rowsPerPage) {
       const numPages = Math.ceil(numRows / rowsPerPage);
+      const showFirstAndLast = this.props.maxButtons ? numPages > this.props.maxButtons : numPages > 10;
```

Comment:
If you want to keep the magic number 10 here, you can shorten to:

`const showFirstAndLast = numPages > (this.props.maxButton || 10);`

or, you could create a default value of 10 for `this.props.maxButton`